Aztec/Classes/TextKit/TextStorage.swift

```diff
@@ -103,6 +114,9 @@ open class TextStorage: NSTextStorage {
         return attachments
     }
 
+
+    // MARK: - Public Helpers
```

Comment:
The word "helper" is extremely generic and not worth much for documentation.  Can you pick a more precise category name for what this section wraps (or use "Misc" if it just needs to be categorized).

In my modest opinion you can just remove this and let this method be part of the "Attachments" section.